merkato/merkato.py

```diff
@@ -9,15 +9,55 @@
 
 
 class Merkato(object):
-    def __init__(self, configuration, ticker, spread):
+    def __init__(self, configuration, ticker, spread, ask_budget, bid_budget):
         self.exchange = Exchange(configuration)
         self.distribution_strategy = 1
         self.ticker = ticker # i.e. 'XMR_BTC'
         self.spread = spread # i.e '15
+        self.ask_profile = [(1, 5), (2, 5), (3, 10), (4, 15), (5, 20), (6, 25), (10, 30)] # (% change in price, % balance allocation)
+        self.bid_profile = [(-1, 5), (-2, 5), (-3, 10), (-4, 15), (-5, 20), (-6, 25), (-10, 30)]  # (% change in price, % balance allocation)
+        self.ask_budget = ask_budget
```

Comment:
why have both a bid and ask budget?

Reply:
Two different currencies? Maybe I'm misunderstanding something.

Reply:
Nasa is correct, every currency on an exchange has two values relative to a Merkato. The first is the total balance in the exchange, and the other is how much is allocated to that Merkato's operation.

Reply:
The bid and ask budgets are how many btc and how many of the alt the user has set aside for bot operation.